Instagram-SERVER/src/main/java/com/sopt/instagram/InstagramSERVER/common/dto/ApiResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.sopt.instagram.InstagramSERVER.common.dto;
+
+import com.sopt.instagram.InstagramSERVER.exception.ErrorStatus;
+import com.sopt.instagram.InstagramSERVER.exception.SuccessStatus;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class ApiResponseDto<T> {

```

Comment:
PR point 첫번째는 다음 어노테이션을 ApiResponseDto 클래스에 붙여주면 해당 값이 null일 때는 프로퍼티를 제외하고 내려줄 수 있습니다 !

```suggestion
@JsonInclude(JsonInclude.Include.NON_NULL)
public class ApiResponseDto<T> {

```

Reply:
오 이런 어노테이션이 있군요?!